tests/run_tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+cd /tests
+
+# Validate hipcc works
+hipcc square.cpp -o square.out --amdgpu-target=gfx801
+ret=$?
+if [ $ret -ne 0 ]; then
+    exit 1
+fi
+
+# Get DNNMark loaded
+git clone https://github.com/doody1986/DNNMark.git
+cd DNNMark/
+git apply ../dnnmark.patch
+./setup.sh HIP
+cd build
+make
+
+# Get the file needed for the DNNMark program
+mkdir -p /.cache/miopen/1.2.0/8cc7cb244c7ad66444adf4fb7d8b94f1/
+cd /MIOpen/src/kernels
+
+# Generate the file
+/opt/rocm/bin/clang-ocl  -DNUM_BATCH=1 -mcpu=gfx801 -Wno-everything MIOpenSoftmax.cl -o /.cache/miopen/1.2.0/8cc7cb244c7ad66444adf4fb7d8b94f1/MIOpenSoftmax.cl.o
```

Comment:
See above comment -- we need a comment here so it doesn't seem like 8cc7... is made up.